server/api.js

```diff
@@ -735,7 +734,7 @@ router.post("/bid", async (req, res) => {
 			}
 			const bidQuery = `
                       INSERT INTO bid (tender_id, bidder_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
-					  VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING bid_id
+					  VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING bid_id
```

Comment:
Please. Test your code.

Reply:
it was working but I have corrected the mistake you pointed to

Reply:
You need to test your code again if you change it